### PR TITLE
[11.0][MIG] account_fiscal_month

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Supersede of https://github.com/OCA/account-financial-tools/pull/636.

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Works good.

**Comment by MiquelRForgeFlow** [comment]
@feketemihai @Cedric-Pigeon @kittiu

File: account_fiscal_month/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Fiscal Month',
+    'summary': """
+        Provide a fiscal month date range type""",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'https://www.acsone.eu',
```
Comment (nikul-serpentcs) [inline account_fiscal_month/__manifest__.py]: https://github.com/OCA/account-financial-tools

**Review by nikul-serpentcs — COMMENTED** [review]
Add README base on https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

**Comment by MiquelRForgeFlow** [comment]
@nikul-serpentcs I won't do the fragmented readme in this PR. FYI, it's already done in https://github.com/OCA/account-financial-tools/pull/894.

**Comment by AaronHForgeFlow** [comment]
@nikul-serpentcs comments attended. Proceeding.

**Review by HviorForgeFlow — APPROVED** [review]

**Review by nikul-serpentcs — APPROVED** [review]
Code LGTM :+1:

**Pull request merged**